channel/channel.go

```diff
@@ -37,7 +37,7 @@ var (
 
 func getPromptPattern() *regexp.Regexp {
 	promptPatternOnce.Do(func() {
-		promptPattern = regexp.MustCompile(`(?im)^[a-z\d.\-@()/:]{1,48}[#>$]\s*$`)
+		promptPattern = regexp.MustCompile(`(?im)^<*[a-z\d.\-@()/:]{1,48}[#>$]\s*$`)
```

Comment:
ah, im guessing this is since we would fail to match in authenticate step without the `<` char? I think we "should" be using the combined pattern from the network driver before open/auth in the channel happens. on driver creation [here](https://github.com/scrapli/scrapligo/blob/321f842f7af185bfda6a73b501e28b521c20b599/driver/network/driver.go#L51) we should be basically joining all the priv levels into one big pattern and setting that on the channel. is this not happening/working right?

if its not working right id like to fix that and then nix adding the `<` here just since this pattern hasn't otherwise changed in a long time (going back to scrapli python early days).

as an aside this pattern is terribly inefficient (the way I set it up, nothing you did!)... but thats a problem for another day...

Reply:
Hi, forgot to mention this change, but this is in order to get the generic driver working, without it it just times out waiting for a prompt that never matches since the huawei prompt has a < in the beginning and not just a > at the end of the prompt.

That said I only really used the generic driver at first before making a driver for VRP so I'm ok with removing it too!